engine/table/src/main/java/io/deephaven/engine/table/impl/locations/util/ExecutorTableDataRefreshService.java

```diff
@@ -87,6 +87,8 @@ private void doRefresh() {
                 refresh();
             } catch (TableDataException e) {
                 subscriptionAggregator.activationFailed(this, e);
+            } catch (Throwable t) {
```

Comment:
Are you sure we want to catch and swallow `Error` here? Maybe better if it was `Exception`?

Reply:
So, I've found that errors are silently dropped if we don't catch them here. Originally, `io.deephaven.engine.table.impl.locations.impl.TableLocationKeySafetyImpl#TableLocationKeySafetyImpl` was throwing IllegalArgument or State exceptions, but from the logs / web UI, it seemed like everything was _ok_. Potentially related to discussions in #3608

Reply:
Silently dropped? Are you certain? It should result in JVM termination. Are we missing an appropriate uncaught exception handler?

Reply:
It might be that we need to actually check for `Error` in the `ScheduledFuture`.

Reply:
Regardless, I agree that we need to catch `Throwable` for now.